### PR TITLE
Docs: add coding guidelines

### PR DESCRIPTION
**Pull request by cwen0**
Signed-off-by: Cwen Yin <cwenyin0@gmail.com> 

Close: #259

### CODE REVIEW

File: docs/developer-guide-overview.md [hunk 1]
```diff
@@ -11,3 +11,8 @@ Then, you can start from [Configure Development Environment](configure-developme
 
 - [Add New Chaos Experiment Type](add-new-chaos-experiment-type.md)
 - [Extend Chaos Daemon interface](extend-chaos-daemon-interface.md)
+
+Coding Guidelines:
```
Comment (iguoyr) [inline docs/developer-guide-overview.md:15]: Maybe `Coding Conventions` is more accurate here.

File: versioned_docs/version-2.1.3/developer-guide-overview.md [hunk 2]
```diff
@@ -11,3 +11,8 @@ Then, you can start from [Configure Development Environment](configure-developme
 
 - [Add New Chaos Experiment Type](add-new-chaos-experiment-type.md)
 - [Extend Chaos Daemon interface](extend-chaos-daemon-interface.md)
+
+Coding Guidelines:
```
Comment (iguoyr) [inline versioned_docs/version-2.1.3/developer-guide-overview.md:15]: ditto

**Review by iguoyr — COMMENTED** [review]
Rest LGTM

File: docs/developer-guide-overview.md (hunk 1 above)
Comment (cwen0) [reply docs/developer-guide-overview.md]: Fixed.

File: versioned_docs/version-2.1.3/developer-guide-overview.md (hunk 2 above)
Comment (cwen0) [reply versioned_docs/version-2.1.3/developer-guide-overview.md]: Fixed:

File: i18n/zh/docusaurus-plugin-content-docs/current/developer-guide-overview.md [hunk 3]
```diff
@@ -11,3 +11,8 @@ title: 开发指南概览
 
 - [新增混沌实验类型](add-new-chaos-experiment-type.md)
 - [拓展 Chaos Daemon 接口](extend-chaos-daemon-interface.md)
+
+开发规范
```
Comment (en-jin19) [inline i18n/zh/docusaurus-plugin-content-docs/current/developer-guide-overview.md:15]:
```suggestion
开发规范相关内容如下：
```

File: i18n/zh/docusaurus-plugin-content-docs/version-2.1.3/developer-guide-overview.md [hunk 4]
```diff
@@ -11,3 +11,8 @@ title: 开发指南概览
 
 - [新增混沌实验类型](add-new-chaos-experiment-type.md)
 - [拓展 Chaos Daemon 接口](extend-chaos-daemon-interface.md)
+
+开发规范
```
Comment (en-jin19) [inline i18n/zh/docusaurus-plugin-content-docs/version-2.1.3/developer-guide-overview.md:15]:
```suggestion
开发规范相关内容如下：
```

Comment (cwen0) [reply i18n/zh/docusaurus-plugin-content-docs/version-2.1.3/developer-guide-overview.md]: Fixed

File: docs/developer-guide-overview.md [hunk 5]
```diff
@@ -11,3 +11,8 @@ Then, you can start from [Configure Development Environment](configure-developme
 
 - [Add New Chaos Experiment Type](add-new-chaos-experiment-type.md)
 - [Extend Chaos Daemon interface](extend-chaos-daemon-interface.md)
+
+Coding Conventions:
```
Comment (en-jin19) [inline docs/developer-guide-overview.md:15]:
```suggestion
For coding conventions, refer to the following documents:
```

File: versioned_docs/version-2.1.3/developer-guide-overview.md [hunk 6]
```diff
@@ -11,3 +11,8 @@ Then, you can start from [Configure Development Environment](configure-developme
 
 - [Add New Chaos Experiment Type](add-new-chaos-experiment-type.md)
 - [Extend Chaos Daemon interface](extend-chaos-daemon-interface.md)
+
+Coding Conventions:
```
Comment (en-jin19) [inline versioned_docs/version-2.1.3/developer-guide-overview.md:15]:
```suggestion
For coding conventions, refer to the following documents:
```

Comment (cwen0) [reply versioned_docs/version-2.1.3/developer-guide-overview.md]: Fixed

**Review by en-jin19 — APPROVED** [review]
LGTM

**Pull request merged**